docker/docker-livecheck.sh

```diff
@@ -3,11 +3,13 @@
 ENV_FILE="${ENV_FILE:-.env}"
 RET_CODE=0
 for service in `docker-compose --env-file=${ENV_FILE} config  --service | tr '\n' ' '`; do 
-if [ -z `docker-compose ps -q $service` ] || [ -z `docker ps -q --no-trunc | grep $(docker-compose --env-file=${ENV_FILE} ps -q $service)` ]; then
-    echo "$service: DOWN"
-    RET_CODE=1
-else
-    echo "$service: UP"
-fi
+    found=$(docker-compose --env-file=${ENV_FILE} ps -- ${service}|grep  ${service}_1|grep '\bUp\b')
```

Comment:
yes, If there was two services, it will fail (because it get two hashes, and grep does not know how to interpret that).

I had the problem because I had a `docker-compose run --rm workers xxx` along (I was doing categorization, and I didn't want to loose my context).
Neither `docker-compose ps` or `docker ps`  do seem to provide any options there (I tried different things).
It seems too me to be a common situation.